mixer/test/client/pilotplugin/pilotplugin_test.go

```diff
@@ -344,9 +347,6 @@ var (
 		Node: &model.Proxy{
 			ID:   "pod2.ns2",
 			Type: model.Sidecar,
-			Metadata: map[string]string{
```

Comment:
why is this metadata removed and not replaced with `NODE_UID` and `NODE_NAMESPACE` ?

Reply:
This is mock metadata to satisfy the plugin interface. It has nothing to do with actual envoy metadata. There is an issue here, that the test combines inbound and outbound routes into one proxy, which is no longer possible with one UID per all configs, but that's for later.